### ISSUE TITLE
Dark Mode doesn't work when using non-standard themes like "Adw-Dark" instead of Adwaita

### ISSUE THREAD

**Issue by DebanjanBarman** [getting-things-gnome/gtg#878]
Dark mode isn't working
![Screenshot from 2022-04-03 08-48-25](https://user-images.githubusercontent.com/46472820/161409834-a2794353-b336-4e83-940e-c00ddb5c4b46.png)
.

**Comment by kocio-pl** [comment]
Try to change system style and give a reply what happens. I have dark style in my system and I have the opposite problem - GTG is always dark, no matter what are the application settings.

**Comment by daniel-sanchez-marzoa** [comment]
Hi, had the same problem but I already have dark style in my system, and gtg is still light

**Comment by Neui** [comment]
What distribution do you use? Do you use some custom style? Try executing GTG with `GTK_DEBUG=interactive` (like `GTK_DEBUG=interactive flatpak run org.gnome.GTG` for the flatpak version, `GTK_DEBUG=interactive gtg` for a locally installed version), and check the Visual tab in the new window.

It should tell you the GTK3 theme (where you can also switch) and whenever darkmode is enabled. Note that the Dark Mode in GTG settings should change that dark mode settings in visual tab, but not the other way around.

Here is me trying it out GTG (flatpak) on Fedora 36 beta live CD in a VM:
![GTG dark mode test in fedora 36 beta](https://user-images.githubusercontent.com/1196130/162439581-769b7552-a3ff-4725-b47c-031cecf3dd4d.png)

**Comment by Chrysostomus** [comment]
I ran into the same issue on Manjaro with the flatpak version. I think I have identified the issue: the gtk-theme we are using is Adw-dark that mimics the libadwaita theme. This makes regular gtk3 apps look like libadwaita apps, but because the corresponing flatpak theme is not installed, gtg does not know what to do and the gtk theme in interactive debugging is unset. 

The solution is to install the corresponding flatpak theme.
